docs/computeds-with-args.md

```diff
@@ -75,7 +75,7 @@ In this specific case the selection could also be stored as an `isSelected` obse
 
 Finally,
 [`computedFn`](https://github.com/mobxjs/mobx-utils#computedfn) from `mobx-utils` can be used in the definition of `todoStore.selected` to automatically memoize `isSelected`.
-It creates a function that memoizes the output for every combination of input arguments.
+It creates a function that memorizes the output for every combination of input arguments.
```

Comment:
It's not misspelled

Reply:
I am pretty sure that "memoization" is a more commonly used term. Memorization is something else.

Reply:
sorry, it's my fault. Do you think it is necessary to merge  grammar corrections?